index.html

```diff
@@ -11,42 +11,44 @@
       <h1 id="page-title">INTENTION TIMER</h1>
     </header>
     <main>
-      <h2 class="activity-title" id="new-activity">New Activity</h2>
-      <section id="user-activity-input">
-        <label class="form-text" id="category">Select a category:</label>
-        <div class="button-box">
-          <div class="activity-card">
-            <button class="activity-button" id="study">
-              <img src="./assets/study.svg">
-              <p class="button-title">Study
-            </button>
+      <section class="big-box">
```

Comment:
Let's make sure to double back on this class attribute and think of something a bit more specific to what it represents!